.github/workflows/build_package.yml

```
@@ -40,19 +40,31 @@ jobs:
         include:
           # Ubuntu packages.
           - runs-on: [managed-releaser, os-family=Linux, runner-group=releaser]
-            build-family: linux
+            build-family: linux-x86_64
+            build-package: main-dist-linux
+            experimental: false
+          - runs-on: [self-hosted, arm64, os-family=Linux, runner-group=releaser]
```

If we want to move forward with postsubmit runner:

```suggestion
          - runs-on: [self-hosted, arm64, os-family=Linux, runner-group=postsubmit]
```

---

I forgot why we'd run the postsubmit runner..? IIUC postsubmit won't run on PR's. Is it for initial stability?

---

I talked to @ScottTodd and we agreed it would be a good idea to set `experimental: true` such that a failing build don't stop fail the job. Also for stability. Let me know what you think.

---

Yes, please. Will be easier to iterate on that way.

---

This workflow only runs on scheduled release jobs or manual triggering. In both cases, it runs on committed patches (or those from folks with write access, which can be more trusted).

---

> I forgot why we'd run the postsubmit runner..? IIUC postsubmit won't run on PR's. Is it for initial stability?

Since this is to build the release package, postsubmit runners are safer to use as presubmit runners might run and access unverified code.

If you want to test the workflow in presubmit, you can change it to `presubmit` temporarily (and it will fail when trying to publish the package). Just need to change it back to postsubmit before merge

---

Pushed `experimental: true` (I removed the commit) and then a patch for using `postsubmit` runner group. Then added a patch `presubmit` to try things out temporarily (will be reverted). I expect it to build fine, since I've tested it locally, but you never know...